packages/core-data/src/selectors.js

```diff
@@ -75,6 +75,17 @@ export const getUserQueryResults = createSelector(
 	( state, queryID ) => [ state.users.queries[ queryID ], state.users.byId ]
 );
 
+/**
+ * Returns all available authors.
+ *
+ * @param {Object} state Data state.
+ *
+ * @return {Array} Authors list.
+ */
+export function getAuthor( state ) {
+	return getUserQueryResults( state, 'author' );
```

Comment:
It might make more sense to use `state.users.byId[ id ]` here.

Reply:
👍

Reply:
This only works if the post author is among the authors in the initial selection. We need to request the post author explicitly.

Reply:
What do you mean? The resolver would still run.

Reply:
the authors resolver gets up to 100 authors. the current post author may not be among them, so  pretty sure in that case `state.users.byId[ id ]` will not be defined?

Reply:
The single author resolver also populates `byId`.

Reply:
ah, ok makes sense. I will change this and verify it works as expected.

Reply:
updated